dvc/external_repo.py

```diff
@@ -158,23 +160,21 @@ def open_by_relpath(self, path, mode="r", encoding=None, **kwargs):
             raise PathMissingError(path, self.url)
 
 
-@wrap_with(threading.Lock())
-def _cached_clone(url, rev):
+def _cached_clone(url, rev, for_write=False):
```

Comment:
Is this `for_write` being used anywhere? Future-proofing?

Reply:
It's needed for write scenarios, which are done in dvcx.